api/envoy/config/overload/v3/overload.proto

```diff
@@ -80,6 +82,31 @@ message Trigger {
   }
 }
 
+message ScaleTimersOverloadActionConfig {
```

Comment:
There are no mentions of this config corresponding to the "envoy.overload_actions.reduce_timeouts" action.

Should envoy.overload_actions.reduce_timeouts be mentioned in this config message comment?
Should there be some doc updates that mention how to configure this action?

Reply:
Done.